carbonserver/carbonserver/api/infra/database/database_manager.py

```diff
@@ -37,23 +35,28 @@ def session(self) -> Callable[..., AbstractContextManager]:
 
         except exc.IntegrityError as e:
             session.rollback()
+            logger.error(e.orig.args[0], exc_info=True)
             raise DBException(
-                error=DBError(code=DBErrorEnum.INTEGRITY_ERROR, message=e.orig.args[0])
+                error=DBError(
+                    code=DBErrorEnum.INTEGRITY_ERROR, message="Relation not found"
+                )
             )
         except exc.DataError as e:
             session.rollback()
+            logger.error(e.orig.args[0], exc_info=True)
             raise DBException(
-                error=DBError(code=DBErrorEnum.DATA_ERROR, message=e.orig.args[0])
+                error=DBError(code=DBErrorEnum.DATA_ERROR, message="Invalid data")
             )
         except exc.ProgrammingError as e:
             session.rollback()
+            logger.error(e.orig.args[0], exc_info=True)
             raise DBException(
                 error=DBError(
-                    code=DBErrorEnum.PROGRAMMING_ERROR, message=e.orig.args[0]
+                    code=DBErrorEnum.PROGRAMMING_ERROR, message="Wrong schema"
```

Comment:
How can I test this? Is there a particular scenario where this exception is thrown?

Reply:
The http validation layer should avoid this to happen if we keep pydantic schemes up to date.
I guess you can try a custom query in a new repository method which tries to insert an incorrect object.

Reply:
I've tested it by adding a not existing field in sql_model.